api/controllers/user.js

```diff
@@ -206,6 +208,46 @@ async function appleLogin(req, accessToken, refreshToken, idToken, profile, done
   return passportLogin(ip, 'apple', accessToken, refreshToken, appleProfile, done);
 }
 
+async function googleIdTokenLogin(req, res) {
+  const client = new OAuth2Client();
+  const id_token = req.query.id_token;
+  async function verify() {
+    const ticket = await client.verifyIdToken({
+      idToken: id_token,
+      audience: [process.env.GOOGLE_FIREBASE_SIGN_IN_APP_ID, process.env.GOOGLE_FIREBASE_WEB_CLIENT_SIGN_IN]
```

Comment:
change `process.env.GOOGLE_FIREBASE_WEB_CLIENT_SIGN_IN` to `process.env.GOOGLE_FIREBASE_WEB_CLIENT_ID` please :)